README.md

```diff
@@ -300,6 +300,9 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [SheSharp](http://www.shesharp.org/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/commons/5/5c/Flag_of_Greece.svg" alt="Greece" width="30">
 
     A Greek community of people dedicated to promoting women's engagement with technology & science. Its main activities consist of free workshops, meetups open to anyone interested, as well as partnerships with other tech communities and local chapters of Thessaloniki.
+    
+  - [Tech Up for Women](https://corporatelearninghub.com/techupforwomen/)
+    Tech Up for Women is a one-day event for the advancement of women in technology through education, resourcing and networking.  The event will focus on new technological advances, cyber security and different financing avenues for startups and privately held companies. This event will provide resources and opportunities for women to improve their digital literacy and successfully compete and disrupt gender imbalance in work environments.  The purpose of the event is to take the fear factor out of technology and empower women to surf the technology wave.
```

Comment:
I think we can remove `to surf the technology wave.` but the rest looks quite great. Good job!